common/src/main/java/com/wynntils/models/beacons/BeaconModel.java

```diff
@@ -23,13 +24,14 @@
 import net.minecraft.world.entity.Entity;
 import net.minecraft.world.entity.EntityType;
 import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.phys.Vec3;
 import net.minecraftforge.eventbus.api.SubscribeEvent;
 import org.apache.commons.compress.utils.Lists;
 
 public class BeaconModel extends Model {
     // Amount of armor stands above each other to consider this a beacon
-    // (A beacon typically has around 17 in total)
-    private static final int VERIFICATION_ENTITY_COUNT = 6;
+    // (A beacon always has 34, we wait for all of them)
```

Comment:
Height is 256, not 128..

Reply:
I'm not sure what you mean by this..?

Reply:
Last time you added that comment by doing 128/7.5, but I misslead you. The height is 256.

Reply:
Aha.